processor/metricstransformprocessor/README.md

```diff
@@ -311,3 +311,29 @@ operations:
   action: group
   group_resource_labels: {"resouce.type": "container", "source": "kubelet"}
 ```
+
+### Attributes Processor for Metrics vs. Metric Transform Processor
```

Comment:
```suggestion
### Metric Transform Processor vs Attributes Processor for Metrics
```
?

Reply:
That is a much better way to word it, I'll update.